src/models/navigation.ts

```diff
@@ -90,6 +90,7 @@ export type ThemedNavigationLogoData = NavigationLogoData & ThemeSupporting<Navi
 export interface HeaderData {
     leftItems: NavigationItemModel[];
     rightItems?: NavigationItemModel[];
+    mobileHeaderItems?: NavigationItemModel[];
```

Comment:
consider using separate field in left/right Items

Reply:
We decided to keep this logic, but rename the name of the prop